lms/routes.py

```diff
@@ -258,6 +258,8 @@ def includeme(config):  # noqa: PLR0915
         factory="lms.resources.dashboard.DashboardResource",
     )
 
+    config.add_route("api.dashboard.courses", "/api/dashboard/courses")
```

Comment:
Now endpoint for listing this. We could in theory reuse one single endpoint and using parameters construct the results, ie:

`?with_assigment_counts=1`

For now I reckon it would be simpler to have one endpoint that returns the results to display in the table an another for listing them for the dropdowns.